src/core_ocean/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -1777,11 +1777,11 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
                                 .or.lat> 10.0.and.lat< 15.0 &
                                 .or.lat> 30.0.and.lat< 35.0 &
                                 .or.lat> 50.0.and.lat< 55.0 ) then
-                              do k = 1, config_reset_debugTracers_top_nLayers
+                              do k = minLevelCell(iCell), minLevelCell(iCell)+config_reset_debugTracers_top_nLayers
                                  tracersGroupNew(3,k,iCell) = 2.0_RKIND
                               end do
                            else
-                              do k = 1, config_reset_debugTracers_top_nLayers
+                              do k = minLevelCell(iCell), minLevelCell(iCell)+config_reset_debugTracers_top_nLayers
```

Comment:
The upper limit of this loop should also have the `-1` appended (like line 1750)